src/main/java/frc/robot/Vision.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot;
+
+import friarLib2.vision.PhotonCameraWrapper;
+import friarLib2.vision.VisionCamera;
+
+/**
+ * Container for the vision systems
+ */
+public class Vision {
+    public static VisionCamera shooterCamera = new PhotonCameraWrapper("");
+
+    /**
+     * @return the distance in meters from the target
+     */
+    public static double getDistanceFromTarget () {
```

Comment:
probably rename ot getMetersFromTarget to be more clear. It's always best when you don't need a comment because the function name is explicit enough. We call this "self documenting code"